integration_tests/steps/device.py

```diff
@@ -82,5 +83,5 @@ def step_impl(context):
 
 @then('the energy bills of the load report the required energy was bought by the load')
 def step_impl(context):
-    assert context.device.latest_stats["device_stats"]["bills"]["bought"] == (24 * 4 - 2) * 0.05
+    assert isclose(context.device.latest_stats["device_stats"]["bills"]["bought"], 22 * 0.2)
```

Comment:
Hmm, I reverted the slot length to 15 mins after our offline discussion. Which one should we chouse?

Reply:
Well, since I tried these locally with 60mins and the tests passed, I am inclined to say to keep the 60 mins slot length, in order to reduce the runtime of the simulation a little.